tgui/packages/tgui/interfaces/NuclearBomb.tsx

```diff
@@ -7,11 +7,13 @@ import {
   NoticeBox,
   ProgressBar,
   Section,
+  Stack,
 } from '../components';
 import { Window } from '../layouts';
 
 type NuclearBombData = {
   status: string;
+  nuke_activation_sites: any;
```

Comment:
Why is this using any? It should use the correct type (string[] i think)

Reply:
Done!